modules/web-console/e2e/testcafe/fixtures/configuration/basic.js

```diff
@@ -34,6 +34,13 @@ fixture('Basic configuration')
     })
     .after(dropTestDB);
 
+test('Check selector picker version list', async(t) => {
+    const page = new PageConfigurationBasic();
+
+    await t.hover(page.versionPicker._selector)
+        .expect(page.versionPicker.menuItems.count).eql(8, 'Should contains 8 supported versions.');
```

Comment:
This assertion looks very strange to me. Why 8? Maybe check something else, like a what version are displayed.

Reply:
Now we have 8 supported versions and do not plain to add some additional.

Reply:
@akuznetsov-gridgain I believe we should check for what version are displayed. Do you agree?

Reply:
Added check of supported versions.